### PR TITLE
New README

### PR DESCRIPTION
**Pull request by swannodette**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,202 +1,66 @@
...
-                 [cljsjs/react-with-addons "0.13.3-0"]]
-  ...)
-```
+(ns example
+  (:require [om.dom :as dom]
+            [om.next :as om]))
 
-### Build configuration
+(defui Widget
+  om/IRender
```
Comment (anmonteiro) [inline README.md]: `Object`

File: README.md [hunk 2]
```diff
@@ -1,202 +1,66 @@
...
+  (render [this]
+    (dom/h1 nil (:text data)))
```
Comment (anmonteiro) [inline README.md]: `(:text (om/props this))`

File: README.md [hunk 3]
```diff
@@ -1,202 +1,66 @@
...
-:cljsbuild {
-  :builds [{:id "dev"
-            :source-paths ["src"]
-            :compiler {
-              :main main.core
-              :output-to "main.js"
-              :output-dir "out"
-              :optimizations :none
-              :source-map true}}]}
+(om/root widget {:text "Hello world!"}
```
Comment (anmonteiro) [inline README.md]: `add-root!`

File: README.md [hunk 4]
```diff
@@ -1,202 +1,66 @@
...
+  {:target (. js/document (getElementById "my-app"))})
 ```
 
-Your markup should look something like the following:
+## Tutorials
```
Comment (anmonteiro) [inline README.md]: I think we can also link to the entire Om Next section in the Wiki?
Link: https://github.com/omcljs/om/wiki#om-next

**Comment by swannodette** [comment]
@anmonteiro thanks! I've made all those changes.

File: README.md [hunk 5]
```diff
@@ -1,202 +1,67 @@
...
-  :builds [{:id "dev"
-            :source-paths ["src"]
-            :compiler {
-              :main main.core
-              :output-to "main.js"
-              :output-dir "out"
-              :optimizations :none
-              :source-map true}}]}
+(om/add-root! widget {:text "Hello world!"}
+  {:target (. js/document (getElementById "my-app"))})
```
Comment (anmonteiro) [inline README.md]: sorry this is still not correct. Needs a reconciler, etc.

**Comment by swannodette** [comment]
@anmonteiro I went ahead and used the simplest case from the Quick Start as an example instead.

**Pull request merged**